src/bp-core/bp-core-functions.php

```diff
@@ -5286,3 +5286,14 @@ function bp_is_admin( $screen_id = '' ) {
 
 	return $screen_id === $bp->admin->current_screen;
 }
+
+/**
+ * Common function to check if bp_classic is active or not.
+ *
+ * @since 15.0.0
+ *
+ * @return bool True if bp_classic is active, false otherwise.
+ */
+function bp_is_classic() {
+	return function_exists( 'bp_classic' ) || ( function_exists( 'is_plugin_active' ) && is_plugin_active( 'bp-classic/bp-classic.php' ) );
```

Comment:
I'm under the impression that @imath suggested to add `is_plugin_active` to this file. Like a polyfill. 🤔 

https://github.com/buddypress/buddypress/blob/master/src/bp-core/bp-core-wpabstraction.php#L190%7CL203

Although, your approach works too. So I guess the question is if we wanna polyfill `is_plugin_active `

Reply:
@imath @renatonascalves Thanks for the feedback, I have implemented the feedback, could you please take the look.